Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -277,5 +277,43 @@ extension BrowserViewController: ReaderModeDelegate {
             println("DEBUG: New readerModeState: \(state.rawValue)")
             toolbar.updateReaderModeState(state)
         }
+
+        // TODO: This is a *temporary* way to trigger the reader mode style dialog via 3 taps in the webview. When
+        // we know where the Aa button needs to go, all code below can be refactored properly.
+        if state == ReaderModeState.Active {
+            let gestureRecognizer = UITapGestureRecognizer(target: self, action: "SELshowReaderModeStyle:")
+            gestureRecognizer.numberOfTapsRequired = 3
+            browser.webView.addGestureRecognizer(gestureRecognizer)
+        } else {
+            if let gestureRecognizers = browser.webView.gestureRecognizers {
+                if gestureRecognizers.count == 1 {
+                    browser.webView.removeGestureRecognizer(gestureRecognizers[0] as UIGestureRecognizer)
```

Comment:
I forgot about the context menu. Either what you suggested or I'll think of something better. I think it is nicer to not have the recognizer hang around all the time.